trustfall_core/test_data/tests/parse_errors/transform_first_directive.graphql.ron

```diff
@@ -0,0 +1,12 @@
+TestGraphQLQuery (
+    schema_name: "filesystem",
+    query: r#"
+{
+    OriginDirectory {
+        out_Directory_Count @transform(op: "count") @output {
```

Comment:
```suggestion
        out_Directory_ContainsFile @transform(op: "count") @output {
```
`out_Directory_Count` is not an edge defined in the schema, so the test might be flaky -- it might report an invalid edge just as well as a failure to use `@transform` correctly.